packages/overdrive/lib/components/NumberInput/NumberInput.tsx

```diff
@@ -11,16 +11,20 @@ const isEdge: boolean =
 
 const type = isEdge ? 'text' : 'number';
 
+const MAX_NUMBER_INPUT_LENGTH= 2147483647;
```

Comment:
```suggestion
const MAX_NUMBER_VALUE= 2147483647;
```
Maybe change this to MAX_NUMBER_INPUT_VALUE